README.md

```diff
@@ -1,5 +1,8 @@
 # Serverless Operator
 
+## Status
+![GitHub tag checks state](https://img.shields.io/github/checks-status/kyma-project/serverless-manager/main?label=serverless-operator&link=https%3A%2F%2Fgithub.com%2Fkyma-project%2Fserverless-manager%2Fcommits%2Fmain)
```

Comment:
Is this a real-time status check of the module? That would be amazing! Can all modules use that functionalitiy? 
Or just an example how that would look like?

Reply:
I've never tested it, but it should be updated automatically.
Yes, all modules can use it.